utils.py

```diff
@@ -332,10 +334,15 @@ def are_datetimes_close(later_datetime, earlier_datetime):
 
 
 def generate_random_string(length):
+    """Generates a random string of the length specified and returns it.
+    Args:
+      length: Length of the string to be generated.
+    """
     return base64.urlsafe_b64encode(os.urandom(length))
```

Comment:
Change it to -- Generates a random string of the specified length.
Write Returns for this function. Also, leave one line before Args and Returns.
Write datatype after length (like-- length: int. Length ...)